# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/wr/WebRenderImageHost.h
# Commit: de6ddf5eb2a7
# Full Hash: de6ddf5eb2a7bd3d8a2f58636760e0aa16c584b9
# Author: Lee Salzman <lsalzman@mozilla.com>
# Date: 2023-12-05 21:29:44
# Regressor Bug: 1829026
# File Overlap Count: 1
# Description:
#   Bug 1829026 - Update CanvasTranslator to work with DrawTargetWebgl. r=aosmond
#   
#   This adds the necessary infrastructure for CanvasTranslator to allocate DrawTargetWebgl
#   instead of just allocating TextureData, and to use RemoteTextureMap to handle sending
#   the DrawTargetWebgl frames to the compositor.
# ==============================================================================

diff -r 773e7b31ac6e -r de6ddf5eb2a7 gfx/layers/wr/WebRenderImageHost.h
--- a/gfx/layers/wr/WebRenderImageHost.h	Tue Dec 05 09:37:03 2023 +0000
+++ b/gfx/layers/wr/WebRenderImageHost.h	Tue Dec 05 09:37:03 2023 +0000
@@ -88,6 +88,7 @@
 
   std::deque<CompositableTextureHostRef> mPendingRemoteTextureWrappers;
   bool mWaitingReadyCallback = false;
+  bool mWaitForRemoteTextureOwner = true;
 
   Maybe<RemoteTextureOwnerId> mRemoteTextureOwnerIdOfPushCallback;
   base::ProcessId mForPidOfPushCallback;